index.html

```diff
@@ -40,6 +40,8 @@ <h2 class="d2l-heading-3">Components</h2>
 			Dropdowns
 			<ul>
 				<li><a href="components/dropdown/demo/dropdown.html">d2l-dropdown</a></li>
+				<li><a href="components/dropdown/demo/dropdown-button.html">d2l-dropdown-button</a></li>
+				<li><a href="components/dropdown/demo/dropdown-button-subtle.html">d2l-dropdown-button-subtle</a></li>
```

Comment:
Nit: I'd be fine with combining these 2 demo pages into a single "openers" or "dropdown-button" one, similar to the tests.